tests/util.go

```diff
@@ -158,3 +160,46 @@ func writeJson(jsonFile string, json string) (string, error) {
 	}
 	return jsonFile, nil
 }
+func runOcDescribeCommand(resourceType, resourceName string) string {
```

Comment:
start with Capital `R` otherwise this keyword will not be visible from other packages.